homeassistant/components/intent/__init__.py

```diff
@@ -0,0 +1,54 @@
+"""The Intent integration."""
+import voluptuous as vol
+
+from homeassistant.core import HomeAssistant
+from homeassistant.components import http
+from homeassistant.components.http.data_validator import RequestDataValidator
+from homeassistant.helpers import config_validation as cv, intent
+
+from .const import DOMAIN
+
+CONFIG_SCHEMA = vol.Schema({DOMAIN: vol.Schema({})}, extra=vol.ALLOW_EXTRA)
+
+
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Set up the Intent component."""
+    hass.http.register_view(ConversationHandleView())
+    return True
+
+
+class ConversationHandleView(http.HomeAssistantView):
```

Comment:
Should the class be renamed to eg `IntentHandleView`?

Reply:
Yes